harness/tests/src/main/kotlin/godot/tests/Entry.kt

```diff
@@ -8,22 +8,22 @@ class Entry : godot.runtime.Entry() {
             registerClass<Invocation>(Invocation::class.qualifiedName!!, "Spatial") {
                 constructor(KtConstructor0(::Invocation))
 
-                property {
+                property(Invocation::x::get, Invocation::x::set, ::KtVariant, KtVariant::asInt) {
```

Comment:
You can pass in a `KMutableProperty` IIRC, so you only need to do `Invocation::x` instead of passing the getter and setter.